scripts/send_email_update.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+import sys
+import lochness
+from pathlib import Path
+import argparse as ap
+import lochness.config as config
+from lochness.email import send_out_daily_updates
+from lochness.transfer import create_s3_transfer_table
+from typing import List
+
+def parse_args(argv):
+    parser = ap.ArgumentParser(description='PHOENIX data syncer')
+    parser.add_argument('-c', '--config', required=True,
+                        help='Configuration file')
+    parser.add_argument('-d', '--days', type=int, default=1,
+                        help='Number of days to summarize dataflow')
+    parser.add_argument('-rd', '--recreate_db', action='store_true',
+                        help='Recreate s3_log database based on the log file')
+    parser.add_argument('-l', '--log_file', default=False,
+                        help='Log file, only used when --recreate_db option '
+                             'is used')
+    parser.add_argument('-r', '--recipients', nargs='+', type=str,
+                        default=False,
+                        help='List of recipients - with this option the '
+                             'recipients in the config file will be ignored')
+    parser.add_argument('-t', '--test', action='store_true',
+                        help='Dry run of email functionality')
```

Comment:
I shall install `lochness` today. Do nothing but run:
> send_email_update.py -r tbillah@bwh.harvard.edu -t

1. Can I run it?
2. If yes, what happens when I run it?

Reply:
In order to send email, you need to add a google account to configuration file and its credentials to the keyring file. Also since your lochness would not have any files transferred to s3, it would send out an email saying that there is no update.